core/templates/exporters/StaticRiverContent.tid

```diff
@@ -1,7 +1,5 @@
 title: $:/core/templates/exporters/StaticRiver/Content
 
-\define renderContent()
-{{{ $(exportFilter)$ ||$:/core/templates/static-tiddler}}}
```

Comment:
hmm, I think, that's not backwards compatible. ... you can't remove `exportFilter` without problems.